scripts/apps/search/directives/ItemRepo.js

```diff
@@ -14,6 +14,16 @@ export function ItemRepo(
         link: function(scope, elem) {
 
             /*
+             * function to initialize default values on init or search provider change
+             */
+            scope.setDefaultValues = function() {
```

Comment:
OK thanks. I'm beginner with angular, so thanks for your advice :), pushing an update soon